xarray/core/indexes.py

```diff
@@ -35,9 +36,6 @@ def __contains__(self, key):
     def __getitem__(self, key):
         return self._indexes[key]
 
-    def __delitem__(self, key):
```

Comment:
undoes my previous change.

 I think I understand the indexes model better now. We only use `._indexes` dict internally and `Indexes` is intended for external use. Is that right?